.github/workflows/pipeline.yml

```diff
@@ -0,0 +1,33 @@
+name: Dockerize API
```

Comment:
Name should not be the commit content. It should reflect what this pipeline actually does

Reply:
Ok, will fix it